cookbooks/private_dependencies.rst

```diff
@@ -0,0 +1,58 @@
+Private Dependencies
+====================
+
+Some projects use private dependencies requiring authentication in order for
+packages to be downloaded. While committing these credentials in the source code
+Git repository source code is technically possible, this is considered as a bad
```

Comment:
double "source code", I would remove both.

Reply:
good catch. And I totally agree. fixed